packages/template-retail-react-app/app/hooks/use-datacloud.js

```diff
@@ -252,6 +287,14 @@ export class DataCloudApi {
                   userDetails
               )
 
+        const partyIdentification = this.dnt
```

Comment:
Not because of the current PR, but on line 283 above, we're returning `null` when dnt. But everywhere else in this file, we return an empty object. Let's make sure it's consistent, thanks.